db/migrate/20170127094357_add_caching_enabled_to_compute_resource.rb

```diff
@@ -0,0 +1,5 @@
+class AddCachingEnabledToComputeResource < ActiveRecord::Migration
+  def change
+    add_column :compute_resources, :caching_enabled, :boolean, default: true
```

Comment:
Won't this make compute resources that don't support caching appear to have caching enabled?